benchmark/CholQRCP_inner_speed_process.cc

```diff
@@ -29,6 +26,7 @@ process_dat() {
     std::vector<std::string> nnz          = {"1"};
     std::vector<std::string> runs         = {"5"};
     std::vector<std::string> num_threads  = {"36"};
```

Comment:
I suspect that these values also need to be specified as integers somewhere in the code. Is that the case? If so, then you should make sure those integer values are based on the values of these things.

Reply:
Not really. Many of those values are only used for properly picking input files and naming output files in the benchmarks that are marked with "Data processing scripts for CholQRCP paper" within benchmark/CMakeLists.txt file. To think about it, those codes should not even be in the /bencmakrs/ folder. They should rather be a part of the "RandLAPACK-test data" repo that I suggested we create, as they really just convert data into different formats.

Reply:
That is not really the case. Most of those values are just use to either find the right input file or name an output file. 
The same goes for other routines that are defined as "data processing scripts" inside of /benchmark/CMakeLists.txt file.

To be honest, those scripts should migrate into a separate repo where we show test result plots and contain test results data. As i mentioned in our slack conversation, it owuld make sense to create a separate "RandLAPACK-benchmark-data" within BALLISTIC LA.

Reply:
Keeping it as is for now per Zoom meeting with Riley.